go/oasis-node/cmd/ias/auth_genesis.go

```diff
@@ -50,6 +50,11 @@ func newGenesisAuthenticator() (iasProxy.Authenticator, error) {
 		logger:   logging.GetLogger("cmd/ias/proxy/auth/genesis"),
 		enclaves: newEnclaveStore(),
 	}
+	for _, rt := range doc.Registry.SuspendedRuntimes {
```

Comment:
Nice catch! We should make sure that the `auth_registry` version works correctly for suspended runtimes as well.

Reply:
Yeah i'm running a test it with auth_registry now, will see if it works.

Reply:
Auth registry works correctly, i actually remember now that i fixed the `WatchRuntimes` awhile ago to include suspended runtimes :-)